romana-install/roles/romana/install-master/tasks/romana_from_github.yml

```diff
@@ -0,0 +1,31 @@
+- name: Download Go
```

Comment:
So, when we use this option, it will never use the default Go version that's installed on my system?

When I run local unit tests I am using my local system install of Go, but for the building here I am using a Go version that prescribed through the script?